### ISSUE TITLE
Generated proto files have full package names

### ISSUE THREAD

**Issue by cuthbe** [alexeyxo/protobuf-swift#150]
###### 3.0
theabyss:generated bencuthbert$ protoc --version
libprotoc 2.6.1
theabyss:generated bencuthbert$
#### ```.proto``` file to reproduce
package com.celertech.staticdata.api.exchange;

import "com/celertech/staticdata/api/enums/AssetTypeProto.proto";

option java_package = "com.celertech.staticdata.api.exchange";

message CreateExchange {

	optional string clientRequestId = 1;

	optional string code = 2;
	optional string name = 3;
	optional string description = 4;
	repeated com.celertech.staticdata.api.enums.assettype.AssetType supportedAssetType = 5;
	optional bool enabled = 6;
	
	repeated CreateExchangeAlias exchangeAlias = 7;
}
#### Description

When I run the generator I get the following output 

Com.Celertech.Staticdata.Api.Exchange.DownstreamExchangeProto.proto.swift

But when that is imported into Xcode it throws an error

'Com' is ambiguous for type lookup in this context

**Comment by lsapan** [comment]
I ran into this issue today. The issue is actually that at the top of the file, it's declaring (in your case) something like:
`public struct com { public struct celertech {public struct staticdata ... }}}` etc

I'm guessing you have multiple files that do this, so the struct `com` is declared multiple times. As such, Swift is confused because `com` shows up multiple times in the global namespace. I'm not super sure this is something that can be fixed in protobuf-swift as you're compiling separate files, but I came up with a quick workaround in the meantime.

Simply take the line I mentioned out of the top of each file, and redeclare it (combine them together) in one place. Problem solved.
